datasets.py

```diff
@@ -2,11 +2,13 @@
 import re
 from rdflib import Graph, ConjunctiveGraph, RDF, Namespace
 from lxltools.datacompiler import Compiler, w3c_dtz_to_ms, last_modified_ms
-from lxltools.contextmaker import DEFAULT_NS_PREF_ORDER, make_context, add_overlay
+from lxltools.contextmaker import make_context, add_overlay
 from urllib.parse import urljoin
 
 
-NS_PREF_ORDER = DEFAULT_NS_PREF_ORDER + ['ldp']
+NS_PREF_ORDER = (
+        'bf2 bf madsrdf skos dc dctype prov sdo bibo foaf void ldp '
```

Comment:
add bflc?

Reply:
Good call! Added in 121eacbcbcfe1a512fe779d81373c790a0e194e1